### PR TITLE
7.x

### PR DESCRIPTION
**Pull request by nhart**
add admin option for default page view. i had to update the pageProgression to get the dual page display to work..seems like we're using pageProgression for something other than what it's intended for.

### CODE REVIEW

**Comment by jonathangreen** [comment]
@nigelgbanks can you take a look at this? It all looks good to me, but I know you were playing with pageProgression.

**Comment by nhart** [comment]
There's an outstanding issue with this pull request. I thought the book reader in sites/all/libraries was our own code. but it turns out it's not. So I need to rethink my approach because I had to fork the ia viewer.

https://github.com/openlibrary/bookreader/blob/master/BookReader/BookReader.js#L47

**Comment by adam-vessey** [comment]
Move https://github.com/Islandora/islandora_internet_archive_bookreader/pull/23/files#L2R17 below https://github.com/Islandora/islandora_internet_archive_bookreader/pull/23/files#L2R19? Seems... inelegant, but it should work. :P

**Comment by adam-vessey** [comment]
Actually... Shouldn't "parent" constructor calls typically be first, anyway?

**Comment by adam-vessey** [comment]
Just looked around a bit... Looks like you can specify the defaults on the object (in the same form as the fragment it uses--which seems weird--but anyway...), instead of setting the mode manually: https://github.com/openlibrary/bookreader/blob/master/BookReader/BookReader.js#L161-L163

File: theme/theme.inc [hunk 1]
```diff
@@ -46,10 +47,11 @@ function islandora_internet_archive_bookreader_preprocess_islandora_internet_arc
         'djatokaUri' => variable_get('islandora_paged_content_djatoka_url', 'http://localhost:8080/adore-djatoka'),
         'imagesFolderUri' => url("$base_url/$library_path/BookReader/images/", array('absolute' => TRUE, 'external' => TRUE)),
         'compression' => variable_get('islandora_internet_archive_bookreader_compression', '4'),
-        'pageProgression' => $variables['page_progression'],
+        'pageProgression' => ($mode == 2 ? 'lr' : $variables['page_progression']),
```
Comment (nigelgbanks) [inline theme/theme.inc:50]: Whats with checking the mode here? Why is it necessary?

File: js/islandora_book_reader.js [hunk 2]
```diff
@@ -15,6 +14,9 @@ function IslandoraBookReader(settings) {
   this.bookUrl = document.location.toString();
   this.imagesBaseURL = settings.imagesFolderUri;
   this.logoURL = '';
+  this.mode = settings.mode
+
+  BookReader.call(this);
```
Comment (nigelgbanks) [inline js/islandora_book_reader.js]: Why was this moved to the bottom of the function?

It seems like it should be at the top otherwise the members values as they are set here will be lost.

File: theme/theme.inc (hunk 1 above)
Comment (nhart) [inline theme/theme.inc:50]: then the mode is set to 2, we want the left and right pane filled. not just the right pane.

File: js/islandora_book_reader.js (hunk 2 above)
Comment (nhart) [inline js/islandora_book_reader.js]: you're right. I initially thought BookReader extended islandora_book_reader, the reverse is true. After switching this back. my fork in libraires/bookreader wasn't necessary.

https://github.com/openlibrary/bookreader/blob/master/BookReader/BookReader.js#L47

File: theme/theme.inc (hunk 1 above)
Comment (nigelgbanks) [inline theme/theme.inc:50]: Well the way I see it is "page_progression" only applies during mode '2'. Forcing it to always be 'lr' removes it as being an option, so why even take the variable 'page_progression'?
